package.json

```diff
@@ -15,7 +15,12 @@
     "htmlparser2": "3.8.3",
     "cssstyle": "0.2.29",
     "webpack": "1.12.2",
-    "webpack-dev-server": "1.12.1"
+    "webpack-dev-server": "1.12.1",
+    "uglify-js": "2.6.1",
+    "bufferutil": "1.2.1",
+    "utf-8-validate": "1.2.1",
+    "tls": "0.0.1",
+    "fs": "0.0.2"
```

Comment:
I thought so too, but they didn't get installed for me by any of the other dependencies. I'll reinstall my environment with the changes discussed in this revision and see if this piece is needed.